German translation fixup

Urshak84 opened the pull request:
#### Checklist

- [x] I have read the [coding conventions](https://github.com/LostArtefacts/TRX/blob/master/CONTRIBUTING.md#coding-conventions)
- [ ] I have added a changelog entry about what my pull request accomplishes, or it is an internal change
- [ ] I have added a readme entry about my new feature or OG bug fix, or it is a different change

#### Description

...

Urshak84 committed:
Add files via upload

Urshak84 committed:
Add files via upload

Urshak84 committed:
Add files via upload

lahm86 commented on data/common/ship/cfg/base_strings-de.json5:
```diff
@@ -98,10 +98,6 @@
         "ENUM_BAR_MODE_FLASHING_OR_DEFAULT": "Standard oder blinkend",
         "ENUM_BAR_MODE_NEVER": "Nie",
         "ENUM_BAR_MODE_PS1": "Standard, kein Blinken",
-        "ENUM_BILLBOARD_LOCK_NONE": "\\{review}Keine",
```
The four entries here will need to be re-added.

lahm86 commented on data/common/ship/cfg/base_strings-de.json5:
```diff
@@ -581,22 +577,20 @@
-        "SETTING_SPRINTBAR_LOCATION_DESCRIPTION": "\\{review}Position, an der die Sprintanzeige angezeigt wird.",
-        "SETTING_SPRINTBAR_SHOW_MODE": "\\{review}Modus der Sprintanzeige",
-        "SETTING_SPRINTBAR_SHOW_MODE_DESCRIPTION": "\\{review}Ändert, wie die Sprintanzeige dargestellt wird.\n\n- Standard: Zeigt die Anzeige während des Sprintens oder der Erholung danach.\n- Nie: Zeigt die Sprintanzeige nie an.",
-        "SETTING_SPRITE_LOCK_MODE": "\\{review}Sprite-Sperrmodus",
```
This will need to be re-added.

lahm86 commented on data/common/ship/cfg/base_strings-de.json5:
```diff
@@ -581,22 +577,20 @@
-        "SETTING_SPRINTBAR_SHOW_MODE": "\\{review}Modus der Sprintanzeige",
-        "SETTING_SPRINTBAR_SHOW_MODE_DESCRIPTION": "\\{review}Ändert, wie die Sprintanzeige dargestellt wird.\n\n- Standard: Zeigt die Anzeige während des Sprintens oder der Erholung danach.\n- Nie: Zeigt die Sprintanzeige nie an.",
-        "SETTING_SPRITE_LOCK_MODE": "\\{review}Sprite-Sperrmodus",
-        "SETTING_SPRITE_LOCK_MODE_DESCRIPTION": "\\{review}Steuert, welche Achsen beim Anzeigen von Sprites auf dem Bildschirm gesperrt werden.\n\n- Keine: Sprites normal anzeigen.\n- Roll: Rollachse sperren – nur im Fotomodus nützlich.\n- Roll & Neigung: Sicherstellen, dass die Sprites aufrecht stehen und nicht auf dem Boden liegen, wenn man sie von oben betrachtet.\n- Perspektive: Roll- und Nei ...
```
And this too.

lahm86 reviewed (changes requested):
Thanks for rebasing your branch. Unfortunately  there are now some missing entries, detailed below.

Also, once you have adjusted an entry you can remove the `\\{review}` part of it, so e.g.
`"GRAPHIC_SETTINGS_UI_TAB": "\\{review}Benutzeroberfläche",`
should become
`"GRAPHIC_SETTINGS_UI_TAB": "Benutzeroberfläche",`

Urshak84 committed:
Add files via upload

The missing strings wherent missing in the file on my pc , i dont know how this could be happening. I hope this file now contains all necessary changes.

Urshak84 commented:
I've tried to reupload the file to fix everyting at once. Did it work ?

lahm86 commented:
> I've tried to reupload the file to fix everyting at once. Did it work ?

I'm afraid not. These entries are still missing.

```
data/common/ship/cfg/base_strings-de.json5: untranslated key '/game_strings/ENUM_BILLBOARD_LOCK_NONE'
data/common/ship/cfg/base_strings-de.json5: untranslated key '/game_strings/ENUM_BILLBOARD_LOCK_PERSPECTIVE'
data/common/ship/cfg/base_strings-de.json5: untranslated key '/game_strings/ENUM_BILLBOARD_LOCK_ROLL'
data/common/ship/cfg/base_strings-de.json5: untranslated key '/game_strings/ENUM_BILLBOARD_LOCK_ROLL_PITCH'
data/common/ship/cfg/base_strings-de.json5: untranslated key '/game_strings/SETTING_SPRITE_LOCK_MODE'
data/common/ship/cfg/base_strings-de.json5: untranslated key '/game_strings/SETTING_SPRITE_LOCK_MODE_DESCRIPTION'
```

Urshak84 committed:
Add files via upload

Urshak84 commented:
I added the missing lines. i hope i didn't miss something.

lahm86 reviewed (approved):
Thanks!
